quickwit-cli/src/index.rs

```diff
@@ -570,22 +611,22 @@ pub async fn ingest_docs_cli(args: IngestDocsArgs) -> anyhow::Result<()> {
         source_type,
         params,
     };
-
-    run_checklist(vec![
-        (
-            "source",
-            Box::pin(check_source_connectivity(&source_config)),
-        ),
-        ("metastore", metastore.check_connectivity()),
-        ("index", metastore.check_index_available(&args.index_id)),
-    ])
-    .await;
-
-    index_metadata.sources = vec![source_config];
+    index_metadata.sources = vec![source_config.clone()];
     let indexer_config = IndexerConfig {
         data_dir_path: args.data_dir_path,
         ..Default::default()
     };
+
+    let checks = vec![
+        ("metastore", Ok(())), // We put it here just to make it nice in the temrinal.
+        ("storage", storage.check().await),
+        ("source", check_source_connectivity(&source_config).await),
+    ];
+    run_checklist(checks);
```

Comment:
@guilload here are the checks :)

Reply:
You can also fake the index with `("index", Ok(()))` :)